engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -371,6 +415,45 @@ namespace dmGameSystem
         dmHashUpdateBuffer32(state, material->m_Textures, sizeof(dmGameSystem::TextureResource*)*material->m_NumTextures);
     }
 
+    static void HashRenderItem(HashState32* state, ModelComponent* component, const MeshRenderItem& item)
```

Comment:
I think we can reuse the render item hashes to solve this issue:
https://github.com/orgs/defold/projects/50?pane=issue&itemId=69469210